mtags/src/main/scala-3/scala/meta/internal/pc/completions/NamedArgCompletions.scala

```diff
@@ -104,12 +116,70 @@ object NamedArgCompletions:
     val params: List[Symbol] =
       allParams.filter(param => param.name.startsWith(prefix))
 
+    val completionSymbols = indexedContext.scopeSymbols
+    def matchingTypesInScope(paramType: Type): List[String] =
+      completionSymbols
+        .collect {
+          case sym
+              if sym.info <:< paramType && sym.isTerm && !sym.info.isNullType && !sym.info.isNothingType && !sym
+                .is(Flags.Method) =>
```

Comment:
[nit] format
```suggestion
              if sym.info <:< paramType &&
                sym.isTerm &&
                !sym.info.isNullType &&
                !sym.info.isNothingType &&
                !sym.is(Flags.Method) =>
```

something like that